doc/design/restore_existing.md

```diff
@@ -0,0 +1,17 @@
+Restore From Existing Backup
+======
+
+We want to restore an etcd cluster from existing backup that operator has saved before.
+
+We add a field in BackupPolicy:
+```Go
+type BackupPolicy struct {
+    // InitPV...
+    InitPV string `json:"initPV"`
```

Comment:
> how should I know what to fill in as the PV name?

When you create cluster "foo", operator will create PVC "foo-pvc-xxx", and the PVC binds to PV "foo-pv-xxx". This relation is something user would know and use to bind to the same PV again.